examples/plantuml/req.py

```diff
@@ -118,7 +118,7 @@ def _print_diagram(fd, diagram):
         expected_dst_path = os.path.join(_out_path, file_dst_path)
         if os.path.isfile(expected_dst_path) is False:
             raise FileNotFoundError( \
-                f"{file_path} diagram name ('@startuml <name>') may differ from file name.")
+                f"{file_path} diagram name ('@startuml <name>') may differ from file name, expected {expected_dst_path}.")
```

Comment:
For consistency, change req2markdown.py accordingly.

Reply:
ok